portfolio-alex.css

```diff
@@ -0,0 +1,232 @@
+*{
```

Comment:
syntax for CSS should be:
```
selector {
    property: value;
}
```

important to get the spaces in the right place so code is consistent